app/lib/frontend/templates/views/shared/layout.dart

```diff
@@ -250,6 +253,7 @@ d.Node pageLayoutNode({
 
 /// Renders the footer content.
```

Comment:
```suggestion
/// Renders the footer content.
///
/// The [canonicalUrl] must be the canonical URL for the current page.
```

Is this correct?

Reply:
Just to clarify that it's not the canonical URL for the content in question.

This is the URL that made the user click "report..." button/link.

Reply:
Yes, that is correct.